package.json

```diff
@@ -8,11 +8,12 @@
     "postinstall": "jetify"
   },
   "dependencies": {
+    "@react-native-async-storage/async-storage": "^1.17.10",
     "@react-native-community/geolocation": "^2.1.0",
     "@react-native-community/push-notification-ios": "^1.10.1",
     "appcenter": "4.4.5",
     "appcenter-crashes": "4.4.5",
-    "bluedot-react-native": "^2.2.0",
+    "bluedot-react-native": "git@github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin.git#ln/enable-bluebar",
```

Comment:
This will be updated to point to dev/release branch of the plugin once the plugin PR has been approved

Reply:
The link can be simplified as `Bluedot-Innovation/Bluedot-React-Native-Plugin#<branch_name>`

Reply:
It seems like there is an issue with that. I have to switch between "git@github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin.git#ln/enable-bluebar" and "Bluedot-Innovation/Bluedot-React-Native-Plugin#<branch_name>" from times to times to avoid skipping integrity check.

Reply:
I have updated the git url to point to release/2.2.1